redis/cache/redispipebp/monitored.go

```diff
@@ -83,34 +105,59 @@ func (s MonitoredSync) Send(ctx context.Context, r redis.Request) (result interf
 		s.Name+".send",
 		tracing.SpanTypeOption{Type: tracing.SpanTypeClient},
 	)
+	command := extractCommand(r.Cmd)
+	if command == "" {
+		command = "send"
+	}
```

Comment:
should we just move this logic into `extractCommand`?

Reply:
updated

Reply:
removed because it breaks the expectation of the other commands